decentraland/unity-renderer
# feat: BiW - Clicking on an entity of the collection hides the collection menu

## What this PR includes?
### Fixes #382

**PROBLEM:** While creating new entities using the Collections menu, it usually happens that entities get created behind the panel.

**SOLUTION:** To avoid this, the panel will become semi-transparent as soon as the player clicks on an entity. This won't be the behaviour while dragging and dropping.

## How to test it?
1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:feat/BiW-Clicking-on-an-entity-of-the-collection-hides-the-collection-menu&ENV=zone&ENABLE_BUILDER_IN_WORLD&position=100,100
2. Press 'K' to open the Builer In World editor.
3. Select items in the catalog.
4. Notice that the entity information panel enters in "transparency mode" to allow you move the item behind.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
